source/routing/raptor_api.cpp

```diff
@@ -1167,73 +1237,144 @@ void make_response(navitia::PbCreator& pb_creator,
                     destination,
                     datetimes,
                     clockwise);
-        if (pb_creator.empty_journeys()) {
-            pb_creator.fill_pb_error(pbnavitia::Error::no_destination,
-                                     pbnavitia::NO_DESTINATION_POINT,
-                                     "no destination point");
+
+        if (pb_creator.empty_journeys())
+        {
```

Comment:
it's a matter of taste but the `{` should be on the same line than the `if`, `for`, etc... and they are required.

Reply:
:white_check_mark: